build.gradle

```diff
@@ -54,17 +55,17 @@ compileJava {
 }
 
 dependencies {
-    compile('org.seleniumhq.selenium:selenium-java:3.4.0'){
+    compile('org.seleniumhq.selenium:selenium-java:3.5.1'){
```

Comment:
@TikhomirovSergey We are good to update to `3.5.2` as `3.5.+`. I don't see any issues on yesterday's selenium release.